src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -429,6 +584,39 @@ def __init__(self, config):
 
         self.config = config
         self.layers = nn.ModuleList()
+        if config.embedding_type == "hybrid":
+            self._init_reassemble_dpt_hybrid(config)
+        else:
+            self._init_reassemble_dpt(config)
+
+        self.neck_ignore_stages = config.neck_ignore_stages
+
+    def _init_reassemble_dpt_hybrid(self, config):
+        r""" "
+        This needs to be re-defined since for `DPTHybrid` the first 2 reassemble layers are set to `nn.Identity()`. For
+        DPT-Hybrid the first 2 reassemble layers are set to `nn.Identity()`, please check the official implementation:
```

Comment:
the same sentence is said twice here

Reply:
Fixed in [7cfd434](https://github.com/huggingface/transformers/pull/20645/commits/7cfd434e50d1a1af01c2387a233c695abbed08b4)